internal/server/instance/drivers/driver_qemu.go

```diff
@@ -2159,7 +2159,7 @@ func (d *qemu) setupNvram() error {
 	d.logger.Debug("Generating NVRAM")
 
 	// Cleanup existing variables.
-	for _, firmwarePair := range edk2.GetAchitectureFirmwarePairs(d.architecture) {
+	for _, firmwarePair := range edk2.GetArchitectureFirmwarePairs(d.architecture) {
```

Comment:
That's not a typo, it's a function which returns pairs of architecture and firmware.